verdictd/app/src/main.rs

```diff
@@ -81,16 +81,88 @@ fn run_server(sockaddr: &str) {
 
 #[tokio::main]
 async fn main() {
-    println!("Verdictd Server Started ...");
+    let version = format!("v{}\ncommit: {}\nbuildtime: {}",
+                    build::PKG_VERSION, build::COMMIT_HASH, build::BUILD_TIME);
+    println!("Verdictd info: {}", version);
+
+    let matches = 
+        App::new("verdictd")
+            .version(version.as_str())
+                .long_version(version.as_str())
+                .author("Inclavare-Containers Team")
+                .arg(Arg::with_name("listen")
+                    .short("l")
+                    .long("listen")
+                    .value_name("sockaddr")
+                    .help("Work in listen mode")
+                    .takes_value(true)
+                )
+                .arg(Arg::with_name("tls")
+                    .long("tls")
+                    .value_name("tls_type")
+                    .help("Specify the TLS type")
+                    .takes_value(true)
+                )
+                .arg(Arg::with_name("crypto")
+                    .long("crypto")
+                    .value_name("crypto_type")
+                    .help("Specify the crypto type")
+                    .takes_value(true)
+                )   
+                .arg(Arg::with_name("attester")
+                    .long("attester")
+                    .value_name("attester_type")
+                    .help("Specify the attester type")
+                    .takes_value(true)
+                )
+                .arg(Arg::with_name("verifier")
+                    .long("verifier")
+                    .value_name("verifier_type")
+                    .help("Specify the verifier type")
+                    .takes_value(true)
```

Comment:
Is it better to to add other related parameters of etls, such as `mutual` and `log-level`?
Although mutual is the default mode. But adding this parameter can help troubleshoot problems during the code development process, for example, you can run through one-way first, and then debug mutual.

Reply:
It's OK to add mutual option. About log-level option, currently verdictd hasn't implememted log level output mechansim. Maybe we can add it later as a separate function.